ft-sdk/migrations/0003-fastn-user-identity-nullable-and-fastn-session-data-non-nullable.sql

```diff
@@ -58,5 +58,5 @@ ALTER TABLE "new__fastn_session" RENAME TO fastn_session;
 
 
 
--- Step 6: Re-enable foreign key constraint enforcement (if necessary)
+-- Re-enable foreign key constraint enforcement (if necessary)
 PRAGMA defer_foreign_keys = OFF;
```

Comment:
You do not have to do this, as it automatically is set to off at the end of transaction.